imaging/models.py

```diff
@@ -3,16 +3,18 @@
 from misc.models import BrainLocation, CoordinateTransformation
 
 class WidefieldImaging(BaseExperimentalData):
+	# we need to talk this through with nick - not sure if he is using multipel files or just one for multispectral
     raw_data = models.ForeignKey(Dataset, blank=True, null=True, related_name="widefield_raw",
                                    help_text="pointer to nT by nX by nY by nC (colors) binary file")
     compressed_data = models.ForeignKey('SVDCompressedMovie', null=True, blank=True, related_name="widefield_compressed",
                                         help_text="Link to SVD compressed movie, if compression was run")
     start_time = models.FloatField(null=True, blank=True,
                                    help_text="in seconds relative to experiment start. TODO: not DateTimeField? / TimeDifference")
+								   # again, all relative to experiment start in seconds.
     end_time = models.FloatField(null=True, blank=True,
                                  help_text="Equals start time if single application. TODO: should this be an offset? Or DateTimeField? Or TimeDifference?")
     imaging_indicator = models.CharField(max_length=255, null=True, blank=True,
-                                         help_text="<GCaMP6f, GCaMP6m, GCaMP6s, VSFPb1.2, intrinsic, …>. TODO: normalize!")
+                                         help_text="<GCaMP6f, GCaMP6m, GCaMP6s, VSFPb1.2, intrinsic, …>. TODO: normalize!") 
```

Comment:
They are generally the same. So yes, a single ImagingIndicators table is what we want.